lib/model/query/submissions.js

```diff
@@ -190,6 +193,10 @@ ${encrypted ? sql`
 left outer join (select id, "keyId" as "encKeyId" from form_defs) as form_defs on form_defs.id=submission_defs."formDefId"
 left outer join (select id, content as "encData" from blobs) as blobs on blobs.id=submission_attachments."blobId"`
     : sql`join (select null as "encKeyId", null as "encData") as enc on true`}
+left outer join
```

Comment:
Could an inner join be used? I think there will always be at least one submission def?